src/NUnitFramework/framework/Constraints/Comparers/StringsComparer.cs

```diff
@@ -31,7 +32,14 @@ public static bool Equals(string x, string y, bool ignoreCase, bool ignoreWhiteS
             }
             else
             {
-                return x.Equals(y, ignoreCase ? StringComparison.CurrentCultureIgnoreCase : StringComparison.Ordinal);
+                IEqualityComparer<string> comparer = (ignoreCase, NormalizeLineEndings) switch
+                {
+                    (true, true) => LineEndingNormalizingStringComparer.CurrentCultureIgnoreCase,
```

Comment:
`CurrentCultureIgnoreCase` and `Ordinal` to be consistent with the existing code.